crates/ty_server/src/document/range.rs

```diff
@@ -165,6 +189,19 @@ impl TextSizeExt for TextSize {
         let source = source_text(db, file);
         let index = line_index(db, file);
 
+        if let Some(notebook) = source.as_notebook() {
```

Comment:
Unrelated to this PR, but there seems to be a some typo in the documentation for `TextSizeExt::to_lsp_position` and `ToRangeExt::to_lsp_range`:

```rs
    /// Converts self into a position into an LSP text document (can be a cell or regular document).
```

Should it be "Converts self into a position for an LSP text ..."?